docs/extending_wsimod.md

```diff
@@ -0,0 +1,95 @@
+# Extending and customising WSIMOD functionality
+
+WSIMOD has many features built-in, but undoubtedly, to model real life scenarios, you will need to customise it to your specific needs.
+
+WSIMOD offers 4 ways of doing this customisation of increased complexity - and flexibility: custom orchestration, overrides, patches and custom classes.
+
+## Custom orchestration
+
+WSIMOD allows to define the order in which the sequence of actions for each node must take place. This orchestration can be customised within the `orchestration` section of the config file. For example, the following orchestration will run the simulation with two actions: first it will run the `infiltrate` method of the `Groundwater` node, and then the `make_discharge` method of the `Sewer` node.
```

Comment:
```suggestion
The model orchestration can be customised within the `orchestration` section of the config file. For example, the following orchestration will run the simulation with two actions: first it will run the `infiltrate` method for all nodes of `type_: Groundwater`, and then the `make_discharge` method for nodes all of `type_: Sewer`. Hint: just because the model sees a node as a given `type_`, doesn't mean the node has to be that class, see the [model tutorial](wsimod_models.md/#nodes) for more detail.
```